src/js/components/projects/detail.tsx

```diff
@@ -242,11 +253,7 @@ const ProjectDetail = (
         breadcrumb={[{ name: project.name }]}
         image={project.repo_image_url}
         sidebar={
-          <div
-            className="slds-m-bottom_x-large
-              metecho-secondary-block
-              slds-m-left_medium"
```

Comment:
@stacyk This fixed the alignment issue you noticed. It means the scratch org header doesn't line up with the project image (if there is one), but I think this is better anyway.